src/main/java/ai/lucidtech/las/sdk/Client.java

```diff
@@ -130,10 +140,14 @@ public JSONObject postDocumentId(String documentId, JSONObject feedback) throws
      *
      * @param consentId Delete documents with this consentId
      * @return Feedback response
-     * @see Client#postDocuments
+     * @see Client#createDocument
      */
-    public JSONObject deleteConsentId(String consentId) throws IOException {
-        HttpUriRequest request = this.createSignedRequest("DELETE", "/consents/" + consentId, new JSONObject());
+    public JSONObject deleteConsent(String consentId) throws IOException {
+        HttpUriRequest request = this.createSignedRequest(
```

Comment:
We are not signing anything anymore, maybe we should change the method name to "createAuthorizedRequest" or something similar.